react/features/subtitles/middleware.ts

```diff
@@ -157,9 +159,13 @@ function _endpointMessageReceived({ dispatch, getState }: IStore, next: Function
 
             // Notify the external API too.
             if (typeof APP !== 'undefined') {
+                const sanitizedTranscriptMessage = { ...newTranscriptMessage };
```

Comment:
Let's please modify this to match the newly added webhook. Sample payload:

```
      "messageID": "018c1abc-de07-77cf-b73e-147bf6340b1b",
      "final": " This",
      "language": "en",
      "participant": {
        "avatarUrl": "https://www.gravatar.com/avatar/583ddb012250f167d239434148feecc0?d=wavatar&size=200",
        "name": "mihdmt",
        "id": "asdfg123"
      }
```

All the necessary fields that are missing should be present in the payload we get as an endpoint message.

Reply:
I knew I forgot something !!!